prowler/lib/outputs/json_ocsf/json_ocsf.py

```diff
@@ -107,6 +107,7 @@ def fill_json_ocsf(finding_output: FindingOutput) -> DetectionFinding:
             ),
             type_uid=DetectionFindingTypeID.Create,
             type_name=DetectionFindingTypeID.Create.name,
+            unmapped_data={"check_id": finding_output.check_id},
```

Comment:
```suggestion
            unmapped={"check_id": finding_output.check_id},
```

Review the format here https://schema.ocsf.io/1.1.0/classes/detection_finding?extensions=

Reply:
Hey! @jfagoagas just fixed py-ocsf-models in this pr: https://github.com/prowler-cloud/py-ocsf-models/pull/32 once it is updated I will apply the change. 
Thanks for the catch!